### ISSUE TITLE
Possibility to use our model for navmesh instead of generating

### ISSUE THREAD

**Issue by AqlaSolutions** [Robmaister/SharpNav#42]
We want our designers to make navmeshes manually but it looks like there is no way to pass it directly to SharpNav without generation step. What can you suggest? Can you add a feature for our case?

**Comment by Robmaister** [comment]
Yeah, this is definitely something that's been on the backlog for a while. Someone else contributed a JSON file format, you could export to that, modify it the way you'd like, then import it for now. Eventually Recast's file format will be implemented as well.

**Comment by AqlaSolutions** [comment]
@Robmaister JSON is not exactly what I'm talking about. I can't open JSON in 3ds Max and use its power to modify the mesh. I would like it to be possible to simply give SharpNav a model tha will be used exactly as it is, without checking climbs, heights, etc.

**Comment by AqlaSolutions** [comment]
@Robmaister hmm, I think climb checking is still necessary.

**Comment by Robmaister** [comment]
What I was trying to say is that you can provide a navmesh via the JSON file format without having to go through the generation process.

The only thing is that you'd have to build the connectivity data yourself or with a tool.

An alternate build process for that kind of a mesh should be available at some point in the future. I just don't have enough bandwidth at the moment to make it a priority
